### PR TITLE
refactor: Simplify tx and validator SimpleTab

### PR DESCRIPTION
**Pull request by ckniffen**
## High Level Overview of Change

- Render the right content using SimpleRow components. This allows more code reuse between the mobile and tablet versions.
- Validator
	- Remove `Updated Date/Time (UTC)` from mobile view.  The value was removed from the desktop view April 2021
	- Change unl url to be white.
	- Remove commas from ledger index. Only place formatted that way
- Transaction
	- Increase account font-size to match other fields.

### Context of Change

This adds more assertions to help with #358 and #452. Was able to get rid of ~100 lines of code.

### Type of Change

- [x] Refactor (non-breaking change that only restructures code)
- [x] Tests (You added tests for code that already exists, or your new feature included in this PR)

### TypeScript/Hooks Update

- [ ] Updated files to React Hooks
- [ ] Updated files to TypeScript

## Before
Before Tx Desktop
Before TX Mobile
Before Validator Desktop
Before Validator Mobile

## After
After Tx Desktop
After Tx Mobile
After Validator Desktop
After Validator Mobile

### CODE REVIEW

**Comment by JST5000** [comment]
For future PR's if there's a bunch of changes, could you pair them up? (Before + After for each page?) - I find it a bit hard to scroll to the right screen when the changes are stacked like this.

**Comment by ckniffen** [comment]
> For future PR's if there's a bunch of changes, could you pair them up? (Before + After for each page?) - I find it a bit hard to scroll to the right screen when the changes are stacked like this.

Will do.

**Review by JST5000 — APPROVED** [review]
LGTM

File: src/containers/Transactions/SimpleTab.jsx [hunk 1]
```diff
@@ -97,37 +72,24 @@ class SimpleTab extends Component {
...
       <div className="simple-body simple-body-tx">
         <div className="rows">
           <Simple type={raw.tx.TransactionType} data={data.summary} />
-          {rowIndex}
+          {width < BREAKPOINTS.landscape && rowIndex}
         </div>
-        {cartIndex}
+        {width >= BREAKPOINTS.landscape && (
+          <div className="index">{rowIndex}</div>
+        )}
```
Comment (mvadari) [inline src/containers/Transactions/SimpleTab.jsx:92]: lol is that really the only difference?

**Comment by mvadari** [comment]
Why is the UNL text changed to be white? Shouldn't it be a link to the UNL?

**Comment by ckniffen** [comment]
> Why is the UNL text changed to be white? Shouldn't it be a link to the UNL?

It was not a link previously.  It just looked like one and used a color slightly different from the link color (`#292` vs `#80ffba`).  I can make it a link if you would like.

**Comment by mvadari** [comment]
> It was not a link previously. It just looked like one and used a color slightly different from the link color (`#229922` vs `#80ffba`). I can make it a link if you would like.

I'm not sure it's necessary for it to be a link, given that a UNL manifest isn't exactly human-readable (hm, maybe we should add a UNL page to the Explorer). That's fine then.

**Review by mvadari — APPROVED** [review]

**Pull request merged**